cypress/integration/seededFlows/adminFlows/pages/landingPage.spec.js

```diff
@@ -124,7 +138,11 @@ describe('Set a landing page from the admin portal', () => {
 
     // Change landing page but then Cancel
     cy.findByRole('main').within(() => {
-      cy.findAllByTestId('private-checkbox').first().check();
+      cy.findAllByRole('checkbox', {
+        name: "Use as 'Locked Screen' Determines if this page will be used as a landing page for anonymous viewers.",
+      })
+        .first()
+        .check();
```

Comment:
Refactor (blocking): Use `cy.findByRole`, not `cy.findAllByRole` as there is only one checkbox.

```suggestion
      cy.findByRole('checkbox', {
        name: "Use as 'Locked Screen' Determines if this page will be used as a landing page for anonymous viewers.",
      })
        .check();
```